tfac/figures/figure2.py

```diff
@@ -3,10 +3,11 @@
 """
 import numpy as np
 import pandas as pd
-from .common import getSetup
-from ..dataImport import form_tensor
-from ..predict import evaluate_accuracy
-from tensorpack import perform_CMTF, calcR2X
+
+from tfac.dataImport import run_CMTF
+from tfac.figures.common import getSetup
```

Comment:
Is there a reason you need to do these all as absolute imports, and not relative?